sys-cluster/teleport/teleport-2.6.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils golang-build systemd user
```

Comment:
where are you using eutils and user eclasses?

Reply:
Remove eutils, user from 2.5.8, 2.6.0, 9999